src/puppetlabs/ezbake/dependency_utils.clj

```diff
@@ -116,6 +117,21 @@
         (assoc acc project-name (.getInputStream jar-file jar-entry)))
       acc)))
 
+(defn get-stream-from-project-jar
+  [lein-project file-path]
+  (let [project-jar-file (fs/file (:target-path lein-project)
```

Comment:
Not sure if there's a more idiomatic way to construct the path to the base project's jar file.  Open to alternate suggestions.